shared_model/validators/any_block_validator.hpp

```diff
@@ -18,6 +19,8 @@ namespace shared_model {
     class AnyBlockValidator {
      public:
       AnyBlockValidator(
+          shared_model::validation::FieldValidator field_validator =
+              shared_model::validation::FieldValidator(),
```

Comment:
If I see correctly, this parameter is used nowhere later?

Reply:
It is needed because SignableModelValidator requires its template validator to have a constructor which accepts field validator